testsuite/features/secondary/min_ssh_push.feature

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Register a salt system to be managed via SSH push
+
+  Scenario: Delete the salt minion for ssh-reverse bootstrap
+    Given I am on the Systems overview page of this "sle_minion"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    And I cleanup minion "sle_minion"
+    Then "sle_minion" should not be registered
+
+  Scenario: Register this minion for SSH push via tunnel
+    Given I am authorized
+    And I go to the bootstrapping page
+    Then I should see a "Bootstrap Minions" text
+    And I enter the hostname of "sle_minion" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I enter "linux" as "password"
+    And I select "1-SUSE-SSH-TUNNEL-DEV-x86_64" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies"
+    And I check "manageWithSSH"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
```

Comment:
Then

Reply:
Not necessarily. This is an action (to wait). But ok.

Reply:
I'll follow Eric here (i.e. keep `And`).